to-do-list/src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+function Button(props) {
```

Comment:
This component never ended up being used. You should either finish the component and use it in the app or remove this file.